components/ui/Tooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+export const Tooltip: React.FunctionComponent<{
```

Comment:
For tooltips I've been using `Tippy`, I'm definitely not opposed to go full custom if you prefer that but can you replace the existing one and remove the dependency on `Tippy`?

Reply:
Tippy is even a better option than my poor component :)